test-ts/manager.test.ts

```diff
@@ -166,28 +206,31 @@ describe("Manager", () => {
 
   describe("#activateGroup()", () => {
```

Comment:
This should be moved in `default-strategy.test.ts` since `activateGroup` is no longer handled by manager contract

Reply:
This is being address in https://github.com/celo-org/staked-celo/pull/109